### ISSUE TITLE
Incorrect morphing of class and disabled attribute

### ISSUE THREAD

**Issue by metametadata** [bigskysoftware/idiomorph#11]
This is the minimal scenario of the issue I encountered using idiomorph with htmx.

The next test fails:

```js
it('issue', function()
{
    let el1 = make('<fieldset id="el"></fieldset>');

    el1.classList.add('foo');
    el1.disabled = true;

    // Also fails (reorder setting class and disabling)
    // el1.disabled = true;
    // el1.classList.add('foo');

    // Also fails (add and remove class)
    // el1.classList.add('foo');
    // el1.classList.remove('foo');
    // el1.disabled = true;

    let el2 = make('<fieldset id="el">hello</fieldset>')

    // Act
    Idiomorph.morph(el1, el2);

    // Assert
    should.equal('hello', el1.innerHTML);
    should.equal(0, el1.classList.length);
    should.equal(false, el1.disabled);
});
```

Result:

> AssertionError: expected false to equal true

**Comment by pbowyer** [comment]
@metametadata Could you try your code using this PR for idiomorph? https://github.com/bigskysoftware/idiomorph/pull/20

**Comment by metametadata** [comment]
Will do.

**Comment by metametadata** [comment]
The test and its variations (see commented out code) pass in the PR 👌

**Comment by 1cg** [comment]
@metametadata this has been fixed in the v0.1.1 release (today) thanks to @pbowyer and @DEfusion (thank you!)
